Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -722,6 +722,103 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - Revoke Access - SGColl", "[.SyncServerCollection]") {
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl; // "sguser:password" base64 encoded
```

Comment:
It seems like the other tests are using collectionPreamble() for bootstrapping the test with some common configuration. The test might also needs an extra setting that isolate each test's docs push to SG. Please check with @jianminzhao.

Reply:
you are right, sendRemoteRequest uses REST API directly. It uses _authHeader, whereas _options is used by Replicator